app/jobs/application_job.rb

```diff
@@ -1,8 +1,10 @@
 class ApplicationJob < ActiveJob::Base
 
-  def with_raven_context(extra)
+  def with_raven_context(extra_context={})
     yield
   rescue => exception
-    Raven.capture_exception(exception, extra: extra)
+    Raven.extra_context(extra_context)
```

Comment:
So I think this is likely the favored API for adding context. I was a bit concerned about thread safety but was relieved to see that DelayedJob clears the Context after each exception: https://github.com/getsentry/raven-ruby/blob/master/lib/raven/integrations/delayed_job.rb#L45. Also since we are only adding the context when we know there is an exception makes the risk even smaller.